Duke.java

```diff
@@ -0,0 +1,216 @@
+import javax.management.InstanceNotFoundException;
+import java.io.*;
+import java.util.ArrayList;
+import java.util.Scanner;
+public class Duke {
+    private String name = "Lakinta";
+    private Scanner scanner = new Scanner(System.in);
+    private ArrayList<String> tasks = new ArrayList<>();
+    private ArrayList<Task> taskArrayList = new ArrayList<>();
+    private int count = 1;
+    private final String filePath = "./data/duke.txt";
+
+    public Duke() {
+        // Load tasks from the file when the chatbot starts up
+        loadTasksFromFile();
+    }
+
+
+    public void greeting() {
+        System.out.println("Hello! I'm " + name +
+                "\nWhat can I do for you?");
+    }
+
+    public void exit() {
+        System.out.println("Bye. Hope to see you again soon!");
+    }
+
+    public void echo() {
+        while (true) {
+            String response = scanner.nextLine();
+            if (response.equals("bye")) {
+                exit();
+                break;
+            } else {
+                System.out.println(response);
+            }
+        }
+        scanner.close();
+    }
```

Comment:
I think this is probably from the previous increments! Can remove if you're not using it anymore just for cleaning upp, similarly for chat() and addTask()